aiida_mlip/calculations/base.py

```diff
@@ -311,6 +313,11 @@ def _add_model_to_cmdline(
             # Raise error if model is None (different than model not given as input)
             if self.inputs.model is None:
                 raise ValueError("Model cannot be None")
-            model_path = self.inputs.model.filepath
+
+            with self.inputs.model.open(mode="rb") as source:
+                with folder.open("modelcopy.model", mode="wb") as target:
```

Comment:
So is `"modelcopy.model`" a temporary copy of the model? Why is it needed and referenced in the tests?

Reply:
it's the file you will actually have in the folder where the calc is actually run. could have a better name

Reply:
But is it cleaned up afterwards, since it goes in `An aiida.common.folders.Folder to temporarily write files on disk.`?

Is this what Giovanni was talking about where this is useful for copying the contents stored in a node for remote submission? So it's making the data in the node actually available to be used, essentially? Or is that different?